packages/app/src/components/Header/Header.tsx

```diff
@@ -86,6 +94,18 @@ const RoutedHeader = ({
                 />
               )
             ),
+            ...spreadIf(
+              isAuthenticated && iModelId && section === "view" && (
```

Comment:
nit, shouldnt this and line 87 also check that projectId is defined as well?

Reply:
Well, the way the router is defined, the projectId "cant" be undefined when the iModelId is undefined because of the structure of the paths, (its /:section/project/:projectId/imodel/:iModelId, (in this case, it will not identify an iModelId if the full URL do not match, and if it does not, then it will not "use" that component for the path.

IE we never have /:section/imodel/:imodelId as a valid path, because of the concept of all our apis, even though technically it would be possible to have only the iModelId to call /imodels API, we need the project in order to be able to "find" the iModel.